controller/src/main/java/org/jboss/as/controller/audit/FileAuditLogHandler.java

```diff
@@ -71,4 +76,19 @@ boolean isDifferent(AuditLogHandler other){
         }
         return false;
     }
+
+    private static boolean getSystemProperty(final String name, boolean defaultValue) {
+        final SecurityManager sm = System.getSecurityManager();
+        String systemProperty;
+        if(sm == null) {
```

Comment:
btw we usually just do "if (System.getSecurityManager() == null) {" since you dont need the reference anyway